cg/apps/hk.py

```diff
@@ -61,27 +61,15 @@ def get_files(self, bundle: str, tags: list, version: int = None):
         """
         return self.files(bundle=bundle, tags=tags, version=version)
 
-    def add_file(self, file, version_obj: models.Version, tag_name, to_archive=False):
+    def add_file(self, file, version_obj: models.Version, tags, to_archive=False):
```

Comment:
the method looks alright. but what about where `add_file` method are used in other places? Will they still send in `tags` as a single string?